File: Example/CZImagePreviewer/ImageCollectionViewCell.m
//
//  ImageCollectionViewCell.m
//  封装套件使用deom
//
//  Created by siu on 2017/5/16.
//  Copyright © 2017年 siu. All rights reserved.
//

#import "ImageCollectionViewCell.h"
#import "UIImageView+WebCache.h"

@implementation ImageCollectionViewCell
- (void)setImageURL:(NSString *)imageURL
{
    _imageURL = imageURL;
    [self.imageView sd_setImageWithURL:[NSURL URLWithString:_imageURL]];
}
@end

File: Example/CZImagePreviewer/ImageCollectionViewCell.swift
//
//  ImageCollectionViewCell.swift
//  ImageCollectionViewCell
//
//  Created by siuzeontou on 2021/9/8.
//  Copyright © 2021 CocoaPods. All rights reserved.
//

import UIKit
import SDWebImage

class ImageCollectionViewCell: UICollectionViewCell {
    @IBOutlet var imageView: UIImageView!
    var imageURL: String? {
        didSet {
            self.imageView.sd_setImage(with: URL(string: imageURL ?? ""))
        }
    }
}
